app/app.go

```diff
@@ -707,6 +707,13 @@ func New(
 		return app.mm.RunMigrations(ctx, app.configurator, fromVM)
 	})
 
+	// NOTE: workaround for the `GetLastCompletedUpgrade` bug
+	// https://github.com/cosmos/cosmos-sdk/issues/11707
+	planNameWorkAround := "v0.7.0-rc3-hotfix-testnet"
```

Comment:
do you need to keep the previous handler ("v0.7.0-hotfix")?

Reply:
`v0.7.0-hotfix` is the plan for the upgrade, but the buggy `GetLastCompletedUpgrade` will return `v0.7.0-rc3-hotfix-testnet` and check if it's registered, so I just add this old plan name as a workaround.